source/base/mpi.cc

```diff
@@ -593,68 +398,24 @@ namespace Utilities
             .size();
         }
       else
-#  endif
-        {
```

Comment:
this code block needs to stay, right?

Reply:
I checked the original code and my code for a few times, and it seems like I didn't remove this block, but instead I added a {} after the else, that might confuse the system, I'm gonna try to remove the {} and see if it helps.